server/rest/src/test/java/org/infinispan/rest/metric/MetricsRestTest.java

```diff
@@ -0,0 +1,38 @@
+package org.infinispan.rest.metric;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.infinispan.client.rest.RestClient;
+import org.infinispan.client.rest.configuration.RestClientConfigurationBuilder;
+import org.infinispan.commons.test.TestResourceTracker;
+import org.infinispan.manager.EmbeddedCacheManager;
+import org.infinispan.rest.helper.RestServerHelper;
+import org.infinispan.test.SingleCacheManagerTest;
+import org.infinispan.test.fwk.TestCacheManagerFactory;
+import org.testng.annotations.Test;
+
+@Test(groups = "unit", testName = "rest.metric.MetricsRestTest")
+public class MetricsRestTest extends SingleCacheManagerTest {
+
+   private RestServerHelper restServer;
+   private RestClient restClient;
```

Comment:
CI is complaining about a thread leaking, I think we need to stop these two.

Reply:
good catch thank you @jabolina